config/routes.rb

```diff
@@ -0,0 +1,77 @@
+Rails.application.routes.draw do
+  # The priority is based upon order of creation: first created -> highest priority.
+  # See how all your routes lay out with "rake routes".
+
+  # You can have the root of your site routed with "root"
+  # root 'welcome#index'
+
+  # Example of regular route:
+  #   get 'products/:id' => 'catalog#view'
+
+  # set root routing
+  root 'tasks#index'
+
+  # set Task routes
+  get '/tasks'              => 'tasks#index', as: 'tasks' # display a list of all tasks
+  get '/tasks/new'          => 'tasks#new'  # return an HTML form for creating a new task
+  post '/tasks'             => 'tasks#create' # create a new task
+  get '/tasks/:id'          => 'tasks#show', as: 'show' # display a specific task
+  get '/tasks/:id/edit'     => 'tasks#edit', as: 'task' # return an HTML form for editing a task
+  patch '/tasks/:id/edit'   => 'tasks#update' # update a specific task
+  delete '/tasks/:id'       => 'tasks#delete', as: 'delete'  # delete a specific task
+  post 'tasks/:id/'         => 'tasks#complete'
+
+
+  # set People Routes
+  get '/people'             => 'people#index', as: 'people'
+  get '/person/:id/tasks'    => 'people#show', as: 'person_detail'
+
```

Comment:
You left comments explaining what each route does -- i wouldn't say that's necessary unless that was something your instructor required or if you feel like the method name is misleading (and if that's the case then work on making the method name make more contextual sense).  The reason for this is imagine future Rowan comes in, or someone else your working with, comes in and refactors a ton of code but forgets to move the comments around or rephrase them - then the comments are move of a hinder than help.  My MO is to aim for super readable code and leave comments as a last resort to help someone read through something particularly tricky.